app/admin-tab/host/controller.js

```diff
@@ -53,24 +49,17 @@ export default Ember.Controller.extend({
         value = ' ';
       }
 
-      model.set('value', value);
+      // until the callback is added to the settings service this is disabled
+      //this.set('saving', true);
 
-      this.set('saving', true);
-      model.save().then(() => {
-        if ( this.get('backToAdd') )
-        {
-          this.transitionToRoute('hosts.new');
-        }
-        else
-        {
-          this.send('goToPrevious');
-        }
-      }).catch((err) => {
-        this.set('errors', [err]);
-      }).finally(() => {
-        this.set('saving', false);
+      Object.keys(model).forEach((item) => {
+        var name = model[item].name.replace(/\./g, C.SETTING.DOT_CHAR).toLowerCase();
```

Comment:
Not sure why this is needed here, the controller should only deal with the `$` variants.  If you do need to transform it, `import { denormalizeName } from 'ui/services/settings` so the logic stays in one place.